iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSerDe.java

```diff
@@ -151,7 +152,23 @@ public void initialize(@Nullable Configuration configuration, Properties serDePr
   private void createTableForCTAS(Configuration configuration, Properties serDeProperties) {
     serDeProperties.setProperty(TableProperties.ENGINE_HIVE_ENABLED, "true");
     serDeProperties.setProperty(InputFormatConfig.TABLE_SCHEMA, SchemaParser.toJson(tableSchema));
+
+    // build partition spec, if any
+    if (serDeProperties.getProperty(serdeConstants.LIST_PARTITION_COLUMNS) != null) {
+      String[] partCols = serDeProperties.getProperty(serdeConstants.LIST_PARTITION_COLUMNS).split(",");
```

Comment:
That's a good question. Elsewhere in the code when parsing the list of columns from the props, they do the same thing and simply split the list by `,`  - but I guess that's no guarantee by itself. I'll try to find out what restrictions there are for column naming.

Reply:
After some digging, it seems that column names can indeed contain `,`. If they do, Hive will use a different delimiter. I've fixed the serDe code to take that into account.

However, right now Iceberg tables do not handle having column names with a comma very well (those tables can be created but can't be read back), so that is something that should be addressed in a separate PR. I've filed a jira for that.